#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_aspell_conv.dpatch by  <rene@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add info on how to convert aspell dicts to hunspell

@DPATCH@
diff -urNad hunspell-1.1.5~/man/hunspell.4 hunspell-1.1.5/man/hunspell.4
--- hunspell-1.1.5~/man/hunspell.4	2007-03-14 14:37:31.000000000 +0100
+++ hunspell-1.1.5/man/hunspell.4	2007-04-11 14:36:27.000000000 +0200
@@ -1023,6 +1023,38 @@
 Hunspell has supported only the first 65536 characters (Basic Multilingual Plane)
 of Unicode Standard, yet.
 
+.SH "Conversion of aspell dictionaries"
+Aspell dictionaries can be easily converted into hunspell. Conversion steps:
+
+\fBdictionary (xx.cwl -> xx.wl):\fR
+
+.nf
+preunzip xx.cwl
+wc -l < xx.wl > xx.dic
+cat xx.wl >> xx.dic
+.fi
+
+\fBaffix file\fR
+
+If the affix file exists, copy it:
+.nf
+cp xx_affix.dat xx.aff
+.fi
+If not, create it with the suitable character encoding (see xx.dat)
+.nf
+echo "SET ISO8859-x" > xx.aff
+.fi
+or
+.nf
+echo "SET UTF-8" > xx.aff
+.fi
+
+It's useful to add a TRY option with the characters of the dictionary with
+frequency order to set edit distance suggestions:
+.nf
+echo "TRY qwertzuiopasdfghjklyxcvbnmQWERTZUIOPASDFGHJKLYXCVBNM" >>xx.aff
+.fi
+
 .SH "SEE ALSO"
 .B hunspell (1),
 .B ispell (1),
